### PR TITLE
Remove tooltip default styles

### PR DESCRIPTION
**Pull request by dennisja**
#### :boom: Breaking Changes

-

#### :rocket: Enhancements

- Enable users to use or remove default tooltip styles

#### :memo: Documentation

- Update tooltip docs

#### :bug: Bug Fix

-

#### :house: Internal

-

Fixes https://github.com/hshoff/vx/issues/650

### CODE REVIEW

**Comment by williaster** [comment]
@dennisja thanks for the PR including tests! 🙌 

It differs slightly from the solution @hshoff discussed/proposed in #650, instead of adding `withDefaultStyles`, can we add `unstyled` (with a default to `= false`), and also update the default of `style` to be `= defaultStyles`? (along with the export of `defaultStyles`) 

this would support the current behavior + `styles` by default, but allow you override all of (or a subset of) `defaultStyles`, or apply no `styles` whatsoever.

**Comment by dennisja** [comment]
I have changed it to behave like how @hshoff suggested but there is an issue with the implementation. If a user accidentally uses the tooltip like 

```jsx
<Tooltip unstyled style={{...someStylesAUserWantsToApply}} />
```
The styles will not be applied

File: packages/vx-tooltip/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@types/classnames": "^2.2.9",
     "@types/react": "*",
-    "@vx/bounds": "0.0.195",
+    "@vx/bounds": "^0.0.195",
```
Comment (williaster) [inline packages/vx-tooltip/package.json:33]: can you actually revert this? version bumps are managed by `lerna` upon release.

**Comment by williaster** [comment]
@dennisja thanks for this contribution! I think that the case you call out is okay, `unstyled` seems like it should be self-explanatory and we can revisit if we get negative feedback about this 👍 

will try and get this out in the `0.0.196` release today.

**Comment by williaster** [comment]
out in `0.0.196`

**Comment by hellosmithy** [comment]
Just a note that this is a breaking change for anyone who was passing in a style prop to Tooltip. Might be worth updating the CHANGELOG to reflect that.

**Comment by hshoff** [comment]
We’ll update the changelog. 

For now, work around is to import defaultStyles and extend it through style prop. 

```jsx
<Tooltip style={{ ...defaultStyle, style }} />
```

**Comment by williaster** [comment]
Updated the [changelog](https://github.com/hshoff/vx/blob/master/CHANGELOG.md#v00196) and included a before/after 👍

**Pull request merged**